src/SurfaceFluxes.jl

```diff
@@ -95,7 +88,6 @@ function Base.show(io::IO, sfc::SurfaceFluxConditions)
     println(io, "C_drag                 = ", sfc.Cd)
     println(io, "C_heat                 = ", sfc.Ch)
     println(io, "evaporation            = ", sfc.evaporation)
```

Comment:
@charleskawczynski `is_converged` flag removed ; now assumes that `error_on_non_convergence() == true`, such that the simulation is interrupted if the convergence criteria aren't met.